### PR TITLE
Add blog post link to dockershim removal faq

### LINKED ISSUE #30975: Add dockershim docs reference along with FAQ to files
cc: @sftim 

Task docs referencing how to check if dockershim removal affects users to be added to the discussions.

Link to be added: (/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes  #30975

### CODE REVIEW

File: content/en/blog/_posts/2020-12-02-dockershim-faq.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ on the deprecation of Docker as a container runtime for Kubernetes kubelets, and
 what that means, check out the blog post
 [Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/).
 
+Also, read this [blog post](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/) to check whether Dockershim deprecation affects you.
```
Comment (sftim) [inline content/en/blog/_posts/2020-12-02-dockershim-faq.md:15]:
```suggestion
Also, you can read [check whether Dockershim deprecation affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/) to check whether it does.
```

Comment (sftim) [reply content/en/blog/_posts/2020-12-02-dockershim-faq.md:15]: The task we're linking to _isn't_ a blog post.

**Comment by reylejano** [comment]
Preview of updated page:
https://deploy-preview-31057--kubernetes-io-main-staging.netlify.app/blog/2020/12/02/dockershim-faq/

/lgtm

**Comment by divya-mohan0209** [comment]
Taking lgtm from [comment](https://github.com/kubernetes/website/pull/31057#issuecomment-999163254) by @reylejano and approving it. 

/approve

**Pull request merged**